### PR TITLE
refactor: update issue tags and add discussion form

### PR DESCRIPTION
**Pull request by brlipi**
## Description of changes
- [x] Move the feature request form from issues to discussions.
- [x] Change contact link of "💬 Questions & Discussions" to point to the "💡 Ideas / Feature requests" category in the repository's discussions.
- [x] Update issue tags to use prefixes.
- [x] Also change existing labels directly on GitHub to add prefixes to them (https://github.com/devopness/devopness/labels)

## Issues resolved by this PR

- [x] [DVN-13404]
- [x] [DVN-13421]
- [x] [DVN-13428]

## Quality Assurance

- Once the changes in this PR are merged and deployed, success criteria is: Feature requests will be moved to the repository's discussions page.

## More info

[DVN-13404]: https://devopness.atlassian.net/browse/DVN-13404?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-13421]: https://devopness.atlassian.net/browse/DVN-13421?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-13428]: https://devopness.atlassian.net/browse/DVN-13428?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
-  - name: 💬 Questions & Discussions
-    url: https://github.com/orgs/devopness/discussions
-    about: Ask questions and discuss ideas and best practices with other Devopness community members
+  - name: 💬 Ideas & Feature Requests
```
Comment (thlmenezes) [inline .github/ISSUE_TEMPLATE/config.yml:3]:
```suggestion
  - name: 💡 Ideas & Feature Requests
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+    url: https://github.com/devopness/devopness/discussions/categories/ideas-feature-requests
+    about: Share your ideas and request features you want to see added to Devopness
```
Comment (thlmenezes) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
    about: Discuss ideas and request features with other Devopness community members
```

**Review by thlmenezes — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
-  - name: 💬 Questions & Discussions
```
Comment (jfoliveira) [inline .github/ISSUE_TEMPLATE/config.yml:3]: @brlipi why is `Questions & Discussions` link being removed? 🤔 
We should keep it.

File: .github/labeler-issue.yml [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
+"scope: web-app":
   - '(Devopness web app)'
-docs:
+"scope: docs":
   - '(Documentation)'
-sdk:
+"scope: sdk":
   - 'SDK-'
-sdk-javascript:
+"scope: sdk-javascript":
```
Comment (jfoliveira) [inline .github/labeler-issue.yml:13]: @brlipi perhaps keep labels sorted alphabetically for easier maintenance and readability?
What do you think?

File: .github/labeler-pr.yml [hunk 5]
```diff
@@ -1,20 +1,20 @@
-repo:
+"scope: repo":
   - "*"
 
-ci:
+"scope: ci":
   - ".github/**/*"
 
-applications:
+"scope: applications":
```
Comment (jfoliveira) [inline .github/labeler-pr.yml:7]: @brlipi perhaps sort labels alphabetically for easier maintenance and readability?

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: .github/labeler-issue.yml (hunk 4 above)
Comment (brlipi) [reply .github/labeler-issue.yml:13]: Labels rearranged.

File: .github/labeler-pr.yml (hunk 5 above)
Comment (brlipi) [reply .github/labeler-pr.yml:7]: Labels rearranged.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 6]
```diff
@@ -1,5 +1,5 @@
...
-    url: https://github.com/orgs/devopness/discussions
-    about: Ask questions and discuss ideas and best practices with other Devopness community members
+  - name: 💡 Ideas & Feature Requests
+    url: https://github.com/devopness/devopness/discussions/categories/ideas-feature-requests
+    about: Discuss ideas and request features with other Devopness community members
```
Comment (thlmenezes) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
    about: Share your ideas and request features you want to see added to Devopness
```

**Review by thlmenezes — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 6 above)
Comment (jfoliveira) [reply .github/ISSUE_TEMPLATE/config.yml:5]: Perhaps make it clear that the link allows two things to be done? `Discuss ideas` OR `request features`?
```suggestion
    about: Discuss ideas or request features you want to see added to Devopness
```

File: .github/labeler-pr.yml (hunk 5 above)
Comment (jfoliveira) [reply .github/labeler-pr.yml:7]: @brlipi I stlll don't see any changes, and no new commits here https://github.com/devopness/devopness/pull/838/commits

File: .github/ISSUE_TEMPLATE/config.yml (hunk 3 above)
Comment (brlipi) [reply .github/ISSUE_TEMPLATE/config.yml:3]: I've replaced it now with two links:

- One for the "help" category
- One for the "general" category

What do you think about it?

File: .github/labeler-pr.yml (hunk 5 above)
Comment (brlipi) [reply .github/labeler-pr.yml:4]: See commit https://github.com/devopness/devopness/pull/838/commits/b6ae1cf50eb69b3fa9987f9655c2c4217375117a

File: .github/ISSUE_TEMPLATE/config.yml (hunk 3 above)
Comment (jfoliveira) [reply .github/ISSUE_TEMPLATE/config.yml:3]: I think that brings too many options for the user that just clicked `New issue`.
Also, when accessing our `Discussions` and click `New discussion` those options will be asked again, so I don't see any advantage on adding more steps to the user to start a discussion.

My vote is to keep two links:
1. Sending users to https://github.com/devopness/devopness/discussions/categories/ideas-feature-requests
    - Possibly named "💡 Ideas & Feature Requests"
2. Sending users to https://github.com/orgs/devopness/discussions
  - `💬 Questions & Discussions`: feel free to propose new icon and title, but it should send users to the home of our `Discussions` tab

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 3 above)
Comment (brlipi) [reply .github/ISSUE_TEMPLATE/config.yml:3]: I've reverted it to the `💬 Questions & Discussions` with the link to the repo's discussions page.

File: .github/labeler-issue.yml [hunk 7]
```diff
@@ -1,14 +1,14 @@
...
   - '(Devopness API)'
-website:
-  - '(Devopness website)'
-web-app:
-  - '(Devopness web app)'
-docs:
+"scope: docs":
   - '(Documentation)'
-sdk:
+"scope: sdk":
```
Comment (thlmenezes) [inline .github/labeler-issue.yml:5]:
```suggestion
"scope: sdks":
```

File: .github/labeler-pr.yml [hunk 8]
```diff
@@ -1,20 +1,20 @@
-repo:
-  - "*"
-
-ci:
+"scope: ci":
   - ".github/**/*"
 
-applications:
+"scope: applications":
```
Comment (thlmenezes) [inline .github/labeler-pr.yml:4]:
```suggestion
"scope: application":
```

File: .github/labeler-pr.yml [hunk 9]
```diff
@@ -1,20 +1,20 @@
...
 
-packages:
+"scope: packages":
   - packages/**/*
 
-sdk:
+"scope: repo":
+  - "*"
+
+"scope: sdk":
```
Comment (thlmenezes) [inline .github/labeler-pr.yml:16]:
```suggestion
"scope: sdks":
```

**Review by thlmenezes — CHANGES_REQUESTED** [review]
Suggestions to fix mismatches in labels

**Review by thlmenezes — APPROVED** [review]

**Review by jfoliveira — APPROVED** [review]

**Pull request merged**